scripts/migration/migration-GPII-3711.js

```diff
@@ -42,7 +42,9 @@ var newValuesForNovaClientCredential = {
         "http://registry.gpii.net/common/language",
         "http://registry.gpii.net/common/DPIScale",
         "http://registry.gpii.net/common/highContrast/enabled",
-        "http://registry.gpii.net/common/selfVoicing/enabled"
+        "http://registry.gpii.net/common/highContrastTheme",
+        "http://registry.gpii.net/common/selfVoicing/enabled",
+        "http://registry.gpii.net/applications/com.microsoft.office.word-ribbon"
```

Comment:
`word-ribbon` is a nested value - does that make a difference to these tests?

Reply:
Very good point. I've adjusted how this preference is defined in various places including defaultSettings.json5.